protocol/x/perpetuals/types/codec.go

```diff
@@ -9,12 +9,16 @@ import (
 
 func RegisterCodec(cdc *codec.LegacyAmino) {
 	cdc.RegisterConcrete(&MsgAddPremiumVotes{}, "perpetuals/FundingSamples", nil)
```

Comment:
Not related to this PR, but... do you know why `"perpetuals/FundingSamples"` here is not `"perpetuals/AddPremiumVotes"`?